src/commands/new.ts

```diff
@@ -57,6 +63,31 @@ export default class New extends Command {
       });
     }
 
+    try {
+      const exampleFiles = await readFile(resolve(__dirname, '../../assets/examples/examples.json'), { encoding: 'utf8' });
+      examples = JSON.parse(exampleFiles);
+    } catch (error) {
+      // no examples found
+    }
+
+    if (examples.length > 0) {
```

Comment:
If flag `--example` is passed, we should not be asking for an example.

```suggestion
    if (!selectedTemplate && examples.length > 0) {
```

Reply:
ah yes, thanks!